packages/@lwc/ssr-compiler/src/compile-template/transformers/text.ts

```diff
@@ -49,8 +46,7 @@ export const Text: Transformer<IrText> = function Text(node, cxt): EsStatement[]
             (!cxt.nextSibling || cxt.nextSibling.type !== 'Text')
     );
     const valueToYield = expressionIrToEs(node.value, cxt);
-    const tempVariable = b.identifier(cxt.getUniqueVar());
 
     cxt.import('htmlEscape');
-    return bYieldEscapedString(tempVariable, valueToYield, isIsolatedTextNode);
+    return [bYieldEscapedString(valueToYield, isIsolatedTextNode)];
```

Comment:
```suggestion
    return [bYieldEscapedString(valueToYield)];
```
Looks like tests are failing because we don't need the extra variable any more.

Reply:
Also we can change the return type of this function to `EsBlockStatement[]` to be a bit more precise.

Reply:
The variable is declared as `export const Text: Transformer<IrText>`, so the return type on the function doesn't actually matter. The return type defined by `Transformer<IrText>` is what will ultimately be used.

Reply:
Thank you